models/intermediate/int_qualtrics__responses.sql

```diff
@@ -44,7 +44,7 @@ response_join as (
         question_response.source_relation
     
     from question_response 
-    join survey_response 
+    join survey_response
```

Comment:
Can we explicitly call out the join we are intending to do here? If this is meant to be an inner join can we explicitly add that.

Reply:
updated to inner